omnibus/config/software/datadog-agent.rb

```diff
@@ -32,6 +32,7 @@
 
   # we assume the go deps are already installed before running omnibus
   command "invoke agent.build --rebuild --use-embedded-libs --no-development", env: env
+  command "invoke systray.build --rebuild --use-embedded-libs --no-development", env: env
```

Comment:
this breaks the linux build (see gitlab pipeline: https://gitlab.ddbuild.io/datadog/datadog-agent/pipelines/214967) and probably the mac build too. I think we should only run this build command on windows